README.rst

```diff
@@ -659,6 +659,11 @@ Choose from the list of available fixers:
                         $strict param. Warning! This
                         could change code behavior.
 
+* **useless_return** [contrib]
```

Comment:
check, done (https://github.com/symfony/symfony/pull/17249)
lets see what comes back before I move the fixer.